README.md

```diff
@@ -0,0 +1,67 @@
+# Static Comp
+Module 2: Solo Challenge
+
+## Table of Contents
+  - [Abstract](#abstract)
+  - [Technologies](#technologies)
+  - [Illustrations](#illustrations)
+  - [Install + Setup](#set-up)
+  - [Contributors](#contributors)
+	- [Wins](#wins)
+	- [Challenges + Improvements](#challenges-+-Improvements)
+  - [Project Specs](#project-specs)
+  - [Deployed Site](#depolyed-site)
+
+## Abstract
+  - This project was a challenge to see how well we could build to match a desktop comp as closely as possible, while still having some creative liberties. 
+
+  - My wireframing of the different device sizes can be found [here](https://miro.com/app/board/o9J_l5Onets=/)
+
+## Technologies
+  - Javascript
+  - HTML
+  - CSS (including normalize.css)
```

Comment:
Because the normalize file is not 100% yours, I'd suggest being explicit about where you got it: Ie this github username at this link